test/osmt-apitest.env.example

```diff
@@ -2,6 +2,6 @@
 # DEBUG=1
 
 # Provide your Okta endpoint and credentials here. This will be used to get a Bearer token for the API tests
-OKTA_URL=https://dev-XXXXXXXX.okta.com
-OKTA_USERNAME=XXX.XXX@example.com
-OKTA_PASSWORD=XXX
+OKTA_URL=https://dev-xxxxxxxx.okta.com
+OKTA_USERNAME=xxx.xxx@example.com
```

Comment:
The grep in the CLI script looks for 6 lc "x" characters. It makes sense to match all patterns in the example file to that.